prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -0,0 +1,248 @@
+EVENTHUB_NAMESPACE_SHARED_ACCESS_KEY="RootManageSharedAccessKey"
+K8S_SECRET_NAME="${EVENTHUB_NAMESPACE_NAME}-overrides"
+K8S_SECRET_NAMESPACE="kyma-installer"
+K8S_SECRET_USERNAME="\$ConnectionString"
+K8S_SECRET_BROKER=""
+K8S_SECRET_PASSWORD=""
+
+#
+# Utility Functions To Make The Actual Cmd Line Calls
+#
+
+createGroup() {
+	shout "Create Azure group"
+	date
+
+	az group create --name "${RS_GROUP}" --location "${REGION}"
+
+	# Wait until resource group will be visible in azure.
+	counter=0
+	until [[ $(az group exists --name "${RS_GROUP}" -o json) == true ]]; do
+		sleep 15
+		counter=$(( counter + 1 ))
+		if (( counter == 5 )); then
+			echo -e "---\nAzure resource group ${RS_GROUP} still not present after one minute wait.\n---"
+			exit 1
+		fi
+	done
+}
+
+# Create The Azure EventHubs Namespace Based On Global Configuration
+cmdCreateEventHubNamespace() {
+  az eventhubs namespace create \
+    --name "${EVENTHUB_NAMESPACE_NAME}" \
+    --resource-group "${RS_GROUP}" \
+    --location "${EVENTHUB_NAMESPACE_LOCATION}" \
+    --sku standard \
+    --enable-kafka true \
+    --enable-auto-inflate true \
+    --capacity "${EVENTHUB_NAMESPACE_MIN_THROUGHPUT_UNITS}" \
+    --maximum-throughput-units "${EVENTHUB_NAMESPACE_MAX_THROUGHPUT_UNITS}" \
+    --output none
+}
+
+# Get The Azure EventHub Namespace Authorization Keys' PrimaryConnectionString
+cmdNamespacePrimaryConnectionString() {
+  az eventhubs namespace authorization-rule keys list \
+    -o json \
+    --resource-group "${RS_GROUP}" \
+    --namespace-name "${EVENTHUB_NAMESPACE_NAME}" \
+    --name "${EVENTHUB_NAMESPACE_SHARED_ACCESS_KEY}" | \
+    jq -r ".primaryConnectionString"
+}
+
+
+# Create The EventHub Namespace Secret And Return It
+cmdCreateventHubNamespaceSecret() {
+
+  IFS="" read -r -d '' kafkaSecret << EOF
+apiVersion: v1
+stringData:
+  brokers: ${K8S_SECRET_BROKER}
+  namespace: ${EVENTHUB_NAMESPACE_NAME}
+  password: ${K8S_SECRET_PASSWORD}
+  username: ${K8S_SECRET_USERNAME}
+kind: Secret
+metadata:
+  name: ${K8S_SECRET_NAME}
+  namespace: ${K8S_SECRET_NAMESPACE}
+  labels:
+    knativekafka.kyma-project.io/kafka-secret: "true"
+    installer: overrides
+    component: knative-eventing-channel-kafka
+    kyma-project.io/installation: ""
+type: Opaque
+EOF
+  echo "$kafkaSecret"
+}
+
+# Verify The Expected Dependencies Are Present On $PATH
+verifyPathDependencies() {
+
+  if ! [[ -x "$(command -v az)" ]]; then
+    shout "Executable 'az' Not Found On \$PATH - Exiting"
+    exit 1
+  fi
+
+  if ! [[ -x "$(command -v jq)" ]]; then
+    shout "Executable 'jq' Not Found On \$PATH - Exiting"
+    exit 1
+  fi
+
+  if ! [[ -x "$(command -v base64)" ]]; then
+    shout "Executable 'base64' Not Found On \$PATH - Exiting"
+    exit 1
+  fi
+
+}
+
+
+function azureAuthenticating() {
+	shout "Authenticating to azure"
+	date
+
+	az login --service-principal -u "${AZURE_SUBSCRIPTION_APP_ID}" -p "${AZURE_SUBSCRIPTION_SECRET}" --tenant "${AZURE_SUBSCRIPTION_TENANT}"
```

Comment:
(nit): let's make it multiline.

```suggestion
	az login \
	    --service-principal \
	    -u "${AZURE_SUBSCRIPTION_APP_ID}" \
	    -p "${AZURE_SUBSCRIPTION_SECRET}" \
	    --tenant "${AZURE_SUBSCRIPTION_TENANT}"
```

Reply:
Changed